rust/Cargo.toml

```diff
@@ -57,7 +57,8 @@ async-trait = "0.1"
 rust-dataframe-ext = []
 datafusion-ext = ["datafusion", "crossbeam"]
 azure = ["azure_core", "azure_storage", "reqwest"]
-s3 = ["rusoto_core", "rusoto_credential", "rusoto_s3", "rusoto_sts", "rusoto_dynamodb", "maplit"]
+s3-native-tls = ["rusoto_core/native-tls", "rusoto_credential", "rusoto_s3/native-tls", "rusoto_sts/native-tls", "rusoto_dynamodb/native-tls", "maplit"]
```

Comment:
how about leaving `s3` feature to native-tls to keep it backwards compatible?

Reply:
:shrug: I don't have a strong opinion either way, but when did we start caring bout backwards compatibility? :laughing:

Reply:
In this case, it's just a simple one line change from our end to avoid unnecessary surprises downstream :) This should also help fix all the builds as well.